pages/volunteer/enrollment.tsx

```diff
@@ -0,0 +1,136 @@
+import { locale } from "@prisma/client";
+
+type VolunteerEnrollmentProps = {
+    session: Record<string, unknown>;
+};
+
+/**
+ * This is the page that a volunteer will use to enroll into a class in a program
+ */
+export const VolunteerEnrollment: React.FC<VolunteerEnrollmentProps> = ({
+    session,
+}: VolunteerEnrollmentProps) => {
+    const router = useRouter();
+    const { classId, page } = router.query;
+    const [pageNum, setPageNum] = useState<number>(
+        page ? parseInt(page as string, 10) : 0,
+    );
+    const { me, isLoading: isMeLoading } = useMe();
+
+    // fetch classInfo from API
+    const { data: classInfoResponse, error: classInfoError } = useSWR(
+        ["/api/class/" + classId, classId, router.locale],
+        fetcherWithId,
+    );
+
+    const isClassInfoLoading = !classInfoResponse && !classInfoError;
+
+    if (isClassInfoLoading || isMeLoading) {
+        return <Loading />;
+    } else if (classInfoError) {
+        return <Box>An Error has occured</Box>;
+    }
+
+    const classInfo = classInfoResponse
+        ? CardInfoUtil.getClassCardInfo(
+              classInfoResponse.data,
+              router.locale as locale,
+          )
+        : null;
+
+    const nextPage = () => {
+        setPageNum(pageNum + 1);
+        window.scrollTo({ top: 0 });
+    };
+
+    const submitVolunteer = async () => {
+        const response = await fetch("/api/enroll/volunteer", {
+            method: "POST",
+            body: JSON.stringify({
+                volunteerId: me.id,
+                classId: classInfo.id,
+            }),
+            headers: {
+                "Content-Type": "application/json",
+            },
+        });
+        const data = await response.json();
+        console.log(data);
+    };
+
+    // render submit page if criminal record not submitted
+    // render update criminal check form if expired
+    const pageElements = !me.volunteer.criminalRecordCheckLink
+        ? [
+              <SubmitCriminalCheckForm onNext={nextPage} />,
+              <MediaReleaseForm onNext={nextPage} />,
+              <ConfirmClassEnrollment
+                  classInfo={classInfo}
+                  onNext={nextPage}
+                  onFinish={submitVolunteer}
+              />,
+          ]
+        : me.volunteer.criminalCheckExpired
+        ? [
+              <UpdateCriminalCheckForm
+                  classInfo={classInfo}
+                  onNext={nextPage}
+              />,
+              <MediaReleaseForm onNext={nextPage} />,
+              <ConfirmClassEnrollment
+                  classInfo={classInfo}
+                  onNext={nextPage}
+                  onFinish={submitVolunteer}
+              />,
+          ]
+        : [
+              <MediaReleaseForm onNext={nextPage} />,
+              <ConfirmClassEnrollment
+                  classInfo={classInfo}
+                  onNext={nextPage}
+                  onFinish={submitVolunteer}
+              />,
+          ];
```

Comment:
The only item that differs here is the first item, We can apply the ternary on the first item only!